docs/why_cran_server.md

```diff
@@ -0,0 +1,41 @@
+* Versioning: Good and convenient versioning is critical when it comes to reproducibility. If team members want to re-run an analysis or if a job is scheduled to run to retrain a model, they shouldn’t have to manually checkout a bunch of git tags across several repositories, or worse – have to find specific untagged old commits. By having a repository of versioned artifacts, reproducibility becomes much more straightforward.
+* Dependency Resolution: Admittedly, Public CRAN [has its faults](https://mran.microsoft.com/timemachine) when it comes to resolving package dependencies. However, `cran-server` allows users to leverage familiar (base R!) tooling to resolve internal dependencies automatically. Doing this using VCS alone would either involve custom tooling or complex `Makefile` installation processes.
+  * Though not necessary, `cran-server` also supports hosting versions of packages from public CRAN. Because many packages only enforce minimum dependency versions, it is often easy to get surprised by breaking changes in external dependencies. By freezing the versions of external packages that your team uses in `cran-server` you can minimize the risk of public package updates breaking your code and instead update carefully on your schedule.
+
+#### Store packages on a networked file system or an FTP server
+As described in the [official documentation](https://cran.r-project.org/doc/manuals/R-admin.html#Setting-up-a-package-repository), it is possible to just use a networked file system or an FTP server. Choosing NFS or FTP, however, requires you or your organization to maintain your own infrastructure. `cran-server` _can_ be run on-prem on a single server and it can also just as easily be run on a cluster or in the cloud using object storage like [Amazon S3](https://aws.amazon.com/s3/) and services such as [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) or [Amazon ECS](https://aws.amazon.com/ecs/). Further, we find releasing packages through the `cran-server` Web API to be a more sustainable system than managing files directly like you would on an FTP server. The API automatically takes care of steps like validating the artifact and updating package manifests, simplifying the process for users and maintainers. 
+
+#### Use a paid service like [Rstudio Package Manager](https://www.rstudio.com/products/package-manager/) 
+Rstudio Package Manager is new and appears to have a few very promising features. Though, it is a young, paid, closed-source solution. `cran-server` has been running internally at Uptake for nearly 2 years, serving thousands of downloads and over 60 packages. And, it’s free, lightweight and [open source](https://github.com/UptakeOpenSource/cran-server)!
+
+#### Use an Artifact repository like [Nexus](https://www.sonatype.com/nexus-repository-sonatype) or [Artifactory](https://jfrog.com/artifactory/)
```

Comment:
Kind of weird to write about a non-solution here. If you were just reading the headers, you'd be confused.

Reply:
this one deserves a note but I agree with Troy that it isn't a solution and feels out of place. Maybe change the header to "write you own integration with an Artifact repository"?

Reply:
updated the header, does that fix the concern?